authutils/utils.go

```diff
@@ -0,0 +1,12 @@
+package authutils
+
+import ()
+
+func elem(searchValue string, collection []string) bool {
```

Comment:
This function is really general purpose beyond auth. Maybe flag it with a TODO to move to a cdisutils package later?

Reply:
Also should we call this contains() or something more descriptive?

Reply:
Both good ideas.

Reply:
Updated in 0c7197e